plugins/google.js

```diff
@@ -26,35 +27,132 @@ hoverZoomPlugins.push({
             '$1'
         );
 
-        // Extract direct links to images using document.scripts data (no api call(s) needed)
-        // Mostly inspired from chocolatebot's userscript : Google DWIMages 
+        // Hook Google 'Open' XMLHttpRequests to catch data & metadata associated with pictures displayed
+        // These requests are issued by client to Google servers in order to obtain new data when user scrolls down
+        // Hooked data is stored in scripts in DOM with class='hoverZoomHookedData'
+        if ($('script.hoverZoomHook').length == 0) { // Inject hook script in document if not already there
+            var hookScript = document.createElement('script');
+            hookScript.type = 'text/javascript';
+            hookScript.text = `if (typeof oldXHROpen !== 'function') { // Hook only once!
+                console.log('Hooking XHROpen');
```

Comment:
Please don't pollute console with log messages. You should use `cLog` instead (defined in hoverzoom.js).

Reply:
ok, done